### DISCUSSION TITLE
Welcome to website-screens Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [Dezenix/website-screens#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * Don't Spam the discussion box.
  * Use of harsh word and disrespect for other will lead to ban

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pratyush-IITBHU** [comment]
Respected Sir,

I am Pratyush, an electrical sophomore at IIT (BHU) Varanasi. I am very much interested in the website-screens project of Dezenix.

I have some knowledge of Python, Javascript , CSS , HTML, and design in Adobe illustrator. So I want to ask how to start and prepare for this project and what prerequisites I need for completing it.

Please kindly inform me about the topics the website is going to cover.

I also request you to please guide me in making my proposal, because I am eagerly interested in this project!

Thank You

Pratyush Singh

**Reply by ghost** [reply]
Hey Pratyush we are glad to have you on team as a contributor for our project.
Refer to the contribution guidelines for the project and join our discord server

**Reply by Pratyush-IITBHU** [reply]
Where can I find the Discord Link??

**Reply by Pratyush-IITBHU** [reply]
Can You also Tell what will be the main goals of the website, because I am writing a proposal for adding ML-related Projects to it, with their documentary?

**Reply by ghost** [reply]
> Where can I find the Discord Link??

https://discord.gg/Psmeb83P

**Reply by ghost** [reply]
> Can You also Tell what will be the main goals of the website, because I am writing a proposal for adding ML-related Projects to it, with their documentary?

Hope into our discord we have a detailed workflow and you can ping me directly there on will have a discussion over it
